vdoctl/cmd/drivers.go

```diff
@@ -68,6 +70,19 @@ var driversCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		ctx := context.Background()
 
+		var vdoConfigList v1alpha1.VDOConfigList
+
+		err := K8sClient.List(ctx, &vdoConfigList)
+		if err != nil {
+			cobra.CheckErr(err)
+		}
+
+		if len(vdoConfigList.Items) > 0 {
+			fmt.Println("vSphere drivers are already configured.")
```

Comment:
Lets say VDO is configured. We currently do not support edit of the configuration.